src/www/ui/admin-license-file.php

```diff
@@ -57,6 +57,7 @@ function RegisterMenus()
   public function Output()
   {
     $V = ""; // menu_to_1html(menu_find($this->Name, $MenuDepth),0);
+    $V .= '<script src="scripts/jquery-3.2.0.min.js"></script>';
```

Comment:
You are importing jquery here (see first screenshot) altough it gets also imported in the footer (see second screenshot).

##### first screenshot:
![2017-07-12_10 14 28](https://user-images.githubusercontent.com/1187050/28108251-2aa29756-66eb-11e7-9507-9e605c970eb8.png)
##### second screenshot:
![2017-07-12_10 12 27](https://user-images.githubusercontent.com/1187050/28108244-2611061e-66eb-11e7-9f4f-8a9b550e099b.png)